index.bs

```diff
@@ -815,6 +815,12 @@ An attribution source is a [=struct=] with the following items:
 :: A [=boolean=].
 : <dfn>fenced</dfn>
 :: A [=boolean=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer, number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/reporting origin=].
```

Comment:
Should this be per reporting origin per destination?

Reply:
Done.